backend/api/Services/IsarService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Api.Services
+{
+    public interface IIsarService
+    {
+        public Task<HttpResponseMessage> StartMission(string missionId);
+        public Task<HttpResponseMessage> StopMission(string missionId);
+    }
+    public class IsarService : IIsarService
+    {
+        private readonly string _isarUrl;
+        private static readonly HttpClient httpClient = new();
+
+        public IsarService(string isarUrl)
+        {
+            _isarUrl = isarUrl;
+        }
+
+        public async Task<HttpResponseMessage> StartMission(string missionId)
+        {
+            string uri = _isarUrl + "/schedule/start-mission";
+
+            uri += $"?ID={missionId}";
```

Comment:
See how the `UriBuilder` is used to build a URI by adding query parameters here: https://github.com/equinor/flotilla/pull/194/files#diff-71f4bc9e9566b67ed462f9df955296cb6abb248cd63f06abfeb34d6a8f363210R95-R102

I think that is a better approach even though it's only one query parameter. Adding those manually is a bit ugly.

Reply:
See my new improved way of providing Querys, if you agree.